ext/pom.xml

```diff
@@ -12,53 +13,56 @@
+  <artifactId>ext-all</artifactId>
+  <packaging>pom</packaging>
+  <name>Narayana: ext</name>
+  <description>Narayana: Ext All</description>
 
-    <properties>
-        <sortpom.skip>true</sortpom.skip>
-    </properties>
+  <properties>
+    <sortpom.skip>false</sortpom.skip>
+  </properties>
 
-    <profiles>
-        <profile>
-            <id>codeCoverage</id>
-            <build>
-                <plugins>
-                    <plugin>
-                        <groupId>org.apache.maven.plugins</groupId>
-                        <artifactId>maven-install-plugin</artifactId>
-                        <executions>
-                            <execution>
-                                <!-- install shaded jacoco ant jar (for use by the QA suite)
-                                     NB the shaded jar is not available via maven
-								-->
-                                <id>copy-jacoco</id>
-                                <phase>validate</phase>
-                                <goals>
-                                    <goal>install-file</goal>
-                                </goals>
-                                <configuration>
-                                    <file>${basedir}/jacocoant.jar</file>
-                                    <groupId>org.jacoco</groupId>
-                                    <artifactId>jacoco-ant</artifactId>
-                                    <version>${version.org.jacoco}</version>
-                                    <packaging>jar</packaging>
-                                </configuration>
-                            </execution>
-                        </executions>
-                    </plugin>
-                </plugins>
-            </build>
-        </profile>
-    </profiles>
+  <profiles>
+    <profile>
+      <id>codeCoverage</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-dependency-plugin</artifactId>
+            <configuration>
+              <artifactItems>
+                <artifactItem>
+                  <groupId>org.jacoco</groupId>
+                  <artifactId>org.jacoco.ant</artifactId>
+                  <version>${version.org.jacoco}</version>
+                  <classifier>nodeps</classifier>
+                  <type>jar</type>
+                  <overWrite>true</overWrite>
+                  <outputDirectory>${basedir}</outputDirectory>
+                  <destFileName>jacocoant.jar</destFileName>
+                </artifactItem>
+              </artifactItems>
+            </configuration>
+            <executions>
+              <execution>
+                <id>copy</id>
+                <goals>
+                  <goal>copy</goal>
+                </goals>
+                <phase>package</phase>
+              </execution>
+            </executions>
+          </plugin>
+        </plugins>
+      </build>
+    </profile>
+  </profiles>
```

Comment:
Is there any reason to split the removal of `maven-install-plugin` and the employment of `maven-dependency-plugin` in two commits? WDYT if we squash these two commits together?

Reply:
There were three separate issues fixed in this PR, hence the reason I've kept them separate.

Reply:
Fair point, thanks for clarifying this :-)